Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM heroku/heroku:16-build as base
+FROM heroku/heroku:18-build as base
 
-ENV PHP_BUILDPACK_VERSION v148
+ENV PHP_BUILDPACK_VERSION v174
```

Comment:
old one complained too much and not sure it even worked properly due EOL of PHP version in 148